packages/core/src/generators/imports.ts

```diff
@@ -181,9 +181,12 @@ export const addDependency = ({
   hasSchemaDir: boolean;
   isAllowSyntheticDefaultImports: boolean;
 }) => {
-  const toAdds = exports.filter((e) =>
-    implementation.includes(e.alias || e.name),
-  );
+  const toAdds = exports.filter((e) => {
+    const petternValues = [e.alias, e.name].filter((p) => p?.length).join('|');
```

Comment:
PatternValues not PetternValues

Reply:
@melloware 
Thank you for review.
I interpreted the name of the variable to be inappropriate. Do you think this is `searchWords' in the sense of the words you want to search for in a regular expression?

Reply:
Yes searchWords would be a good name

Reply:
Thanks! I fixed it with bellow:

https://github.com/anymaniax/orval/compare/9f349c41c88d6e17162d81c087ff6dc59f2d6718..5d01571013e61fc45575874279201c3c9736746a